src/Microsoft.TemplateEngine.TemplateLocalizer.Core/TraversalArgs.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Collections.Generic;
-using Microsoft.Extensions.Logging;
```

Comment:
Why can this be removed? Don't we have analyzer that errors if unused using is present?

Reply:
My feeling is analyzer is not run on files/projects that were not changed. And this time I ran analyzer via "run analysis on solution"